package com.example.blogsapp.presentation.navigation

import kotlinx.serialization.Serializable

@Serializable
sealed interface Route {
    @Serializable
    data object BlogListScreen : Route
    @Serializable
    data class BlogContentScreen(val blogId: Int) : Route
}